src/odemis/gui/cont/streams.py

```diff
@@ -3282,7 +3282,13 @@ def __init__(self, name, roc, colour, tab_data, project_panel, view_ctrl):
         self._project_panel = project_panel
         self._view_ctrl = view_ctrl
 
-        self.model = odemis.acq.fastem.FastEMROA(name, acqstream.UNDEFINED_ROI, roc)
+        self._asm = self._tab_data.main.asm
```

Comment:
Why do you make them (private) attributes? I don't think we need them anywhere else in this class, or do you plan on extending it?